src/decoders/base64_decoder.rs

```diff
@@ -77,28 +85,53 @@ fn decode_base64_no_error_handling(text: &str) -> Option<String> {
 #[cfg(test)]
 mod tests {
     use super::Base64Decoder;
-    use crate::decoders::interface::{Crack, Decoder};
+    use crate::{
+        checkers::{
```

Comment:
This is cool syntax I haven't seen before!

Reply:
`cargo fmt` does all the magic <3